modules/cardano-production.nix

```diff
@@ -0,0 +1,15 @@
+with (import ./../lib.nix);
+
+params:
+{ name, config, pkgs, resources, ... }: {
+  imports = [
+    ./production.nix
+    ./datadog.nix
+    ./papertrail.nix
+  ];
+
+  global.dnsHostname = if params.typeIsRelay then "cardano-node-${toString params.relayIndex}" else null;
```

Comment:
It _does_ look like `typeIsRelay` is at least a good match here.

What are other possible good ways to state the same thing?